docs/usage/configuration-options.md

```diff
@@ -2044,6 +2044,9 @@ Consider using instead `matchCurrentValue` if you wish to match against the raw
 }
 ```
 
+The syntax of the SemVer range must follow the [versioning scheme](https://docs.renovatebot.com/modules/versioning/#supported-versioning)
+used by the [manager](https://docs.renovatebot.com/modules/manager/#supported-managers) which discovered the configured package.
```

Comment:
versioning can be overwritten by package rule after extraction

Reply:
> The syntax of the SemVer range must follow the [versioning scheme](https://docs.renovatebot.com/modules/versioning/#supported-versioning)
used by the matched packages. This is usually defined by the [manager](https://docs.renovatebot.com/modules/manager/#supported-managers) which discovered them.

?

Reply:
sure but change semver to version, other versioning are not semver.